Matcher.js

```diff
@@ -20,8 +20,18 @@ function Matcher() {
         // An array containing all workshop objects sorted from least to most popular
         this.workshopsByPopularity = [];
```

Comment:
I would assume this variable name would sort from highest popularity to lowest, would it be better practice to rename this something like "workshopsByUnpopularity" or "workshopsByLowPopularity"?